util/dvsim/SynCfg.py

```diff
@@ -0,0 +1,372 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+r"""
+Class describing synthesis configuration object
+"""
+
+import logging as log
+import sys
+from pathlib import Path
+
+from tabulate import tabulate
+
+from Deploy import *
+from Modes import *
+from OneShotCfg import OneShotCfg
+from utils import *
+
+
+class SynCfg(OneShotCfg):
+    """Derivative class for linting purposes.
+    """
+    def __init__(self, flow_cfg_file, proj_root, args):
+        super().__init__(flow_cfg_file, proj_root, args)
+
+    def __post_init__(self):
+        super().__post_init__()
+        # Set the title for lint results.
+        self.results_title = self.name.upper() + " Synthesis Results"
+
+    @staticmethod
+    def create_instance(flow_cfg_file, proj_root, args):
+        '''Create a new instance of this class as with given parameters.
+        '''
+        return SynCfg(flow_cfg_file, proj_root, args)
+
+    def gen_results_summary(self):
+        '''
+        Gathers the aggregated results from all sub configs
+        '''
+
+        # Generate results table for runs. Note that we build a simple md and
+        # a marked up md version here in parallel.
```

Comment:
Which one is the marked up version?

Reply:
that comment looks stale. I think even the lint class still has it. I'll remove it in both places.